package.json

```diff
@@ -23,9 +23,9 @@
     "lock": "npm run lint:versions -s && npm shrinkwrap --dev && sed -i '' 's/https:/http:/g' npm-shrinkwrap.json"
   },
   "dependencies": {
-    "jquery": "^2.2.4",
-    "uikit": "3.0.0-beta.23",
-    "throttle-debounce": "^1.0.1"
+    "jquery": "2.2.4",
+    "uikit": "3.0.0-beta.27",
+    "throttle-debounce": "1.0.1"
```

Comment:
Like Greenkeeper or something like that? I'd be ok with that. I haven't encountered anything that would make make me want to avoid using the latest `npm` yet, personally. I'd say the faster (allegedly) installs and lockfile by default would be reasons _to_ upgrade.